liiweb/templates/liiweb/_legislation_nav.html

```diff
@@ -0,0 +1,36 @@
+{% if locality_type == "province" %}
+  <nav class="nav nav-tabs mb-3 border-bottom">
```

Comment:
The two `if` blocks are identical except for whether subleg is included. Rather remove the if block, and then pass in a flag from the View object to indicate whether the subleg should be shown.